apps/prairielearn/src/tests/sync/assessmentsSync.test.ts

```diff
@@ -2505,4 +2505,61 @@ describe('Assessment syncing', () => {
     assert.equal(syncedData.alternative_groups[0].json_has_alternatives, false);
     assert.equal(syncedData.alternative_groups[1].json_has_alternatives, true);
   });
+
+  it('syncs advanceScorePerc correctly', async () => {
+    const courseData = util.getCourseData();
+    const assessment = makeAssessment(courseData, 'Homework');
+    assessment.advanceScorePerc = 50;
+    assessment.zones = [
+      {
+        title: 'zone 1',
+        questions: [
+          {
+            id: util.QUESTION_ID,
+            points: 1,
+          },
+        ],
+      },
+      {
+        title: 'zone 2',
+        advanceScorePerc: 60,
+        questions: [
+          {
+            id: util.ALTERNATIVE_QUESTION_ID,
+            points: 1,
+            advanceScorePerc: 70,
+          },
+          {
+            advanceScorePerc: 80,
+            alternatives: [
+              {
+                id: util.MANUAL_GRADING_QUESTION_ID,
+                points: 1,
+              },
+              {
+                id: util.WORKSPACE_QUESTION_ID,
+                points: 1,
+                advanceScorePerc: 90,
+              },
+            ],
+          },
+        ],
+      },
+    ];
+    courseData.courseInstances[util.COURSE_INSTANCE_ID].assessments['newhomework'] = assessment;
```

Comment:
nit: can we pull out `newhomework` into a variable

Reply:
This is mirroring every other test that already exists, both in this file and in the other syncing tests. You're welcome to take another pass in a separate PR to change them all at once.

Reply:
Sounds good.